src/main/resources/application.conf

```diff
@@ -26,6 +26,9 @@ lorre: {
   purgeAccountsInterval: 10
   #Used to select how many fees should be averaged together
   numberOfFeesAveraged: 1000
+  #read the tezos net to use from env or default
+  tezos-network: "zeronet"
+  tezos-network: ${?LORRE_TEZOS_NETWORK}
```

Comment:
What is this line supposed to do?

Reply:
it first assigns the value `"zeronet"` to `tezos-network`
the it _tries_ to ovverride it by reading from the environment a variable named `LORRE_TEZOS_NETWORK`, and setting that to the config key _only the env-var is set_

see [here](https://github.com/lightbend/config#optional-system-or-env-variable-overrides)

Reply:
Could we read this as a command line argument instead of an env var?

Reply:
sure, will do